tests/autzen.rs

```diff
@@ -90,6 +103,13 @@ fn test_seek_0_works_on_laz() {
     test_seek_0_works_on("tests/data/autzen.laz");
 }
 
+// TODO: This test hangs in LasZipDecompressor::seek
```

Comment:
Do we have a guess why this is?

Reply:
I didn't debug it any further, but I found a similar issue reported https://github.com/laz-rs/laz-rs/issues/46.